src/components/InnerForm/InnerForm.tsx

```diff
@@ -21,9 +27,13 @@ const InnerForm: React.FC<InnerFormProps> = (props) => {
 
     if (isYandexDataForm(formData)) {
         const {onLoad, ...rest} = formData.yandex;
+
+        const contextProps = formsConfig as YandexFormsContextProps;
+
         return (
             <div className={className}>
                 <YandexForm
+                    {...contextProps}
```

Comment:
Can we do it right here? `...formsConfig as YandexFormsContextProps`

Reply:
done